src/test/java/com/puppycrawl/tools/checkstyle/api/AbstractCheckTest.java

```diff
@@ -378,6 +378,19 @@ public void testAstLog() throws Exception {
                 .isEqualTo(5);
     }
 
+    @Test
+    public void testGetFileName() {
+        final String fileName = "myFile.java";
+        final FileContents fileContents = new FileContents(new FileText(
+            new File(fileName),
+            Collections.singletonList("  //   ")));
+        final DummyAbstractCheck check = new DummyAbstractCheck();
+        check.setFileContents(fileContents);
+        assertWithMessage("Should return true when in package info")
+            .that(check.getFileName())
+            .isEqualTo(fileName);
```

Comment:
why we need this unit test ?
it should be covered by all other tests.

Reply:
same as https://github.com/checkstyle/checkstyle/pull/11502#discussion_r841403816